cypress/support/api/blocks.js

```diff
@@ -0,0 +1,19 @@
+Cypress.Commands.add('postBlock', (block) => {
```

Comment:
mb names like createBlockHttp, createRequestHttp are better

Reply:
or createRequestApi - to avoid overrides by commands that use interactors like createCreate (via form)

Reply:
I'm not sure I have a clear vision of the command naming here.
For example we use aforementioned `putRequest` to actually cancel the request.
But if I name the command `cancelRequest`, it could not be reused for another possible Request modifications (which is PUT for)